### ISSUE TITLE
table header disappears in production!

### ISSUE THREAD

**Issue by luanphungba** [ant-design/ant-design#23023]
- [ ] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.
error demo:
![image](https://user-images.githubusercontent.com/43276516/78744248-4e579880-798b-11ea-896a-51edecbdbb72.png)

my code:
![image](https://user-images.githubusercontent.com/43276516/78744285-629b9580-798b-11ea-8ac5-9c739fcf14bd.png)
note that: i leave one column at least without width to fit fluid layout as document mentions.

### Reproduction link
[![Edit on CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/antd-reproduction-template-pch6q)

### Steps to reproduce
- choose fixed header and render about 300 rows.
- you will see the header appears slower than rows (refresh browser on CodeSandbox too see).
- when i build my app, the header disappears.

### What is expected?
the header appear and fixed or sticky in production when display large amounts of data.

### What is actually happening?
the header disappears in production

| Environment | Info |
|---|---|
| antd | 4.0.0 |
| React | 16.9.0 |
| System | window 10 |
| Browser | chrome |

---
this error appears when display large amounts of data. My app need render 300 rows per page so i want the header shows in production tooo

**Comment by luanphungba** [comment]
i added reproduction link!

**Comment by luanphungba** [comment]
i fixed this issue, close!

**Comment by afc163** [comment]
How did you resolve it, could you share it?

**Comment by johngarcia9110** [comment]
@luanphungba I am also dealing with this issue. What did you do to fix it?

**Comment by lujiajian666** [comment]
@luanphungba how did you fix it ?

**Comment by Nxtra** [comment]
I found a way to mitigate the issue, although it's not the solution for the base of the problem.

In state I keep a tableKey: `const [tableKey, setTableKey] = useState(uuidV4());`

I added the property `key={tableKey}` to my `Table` component.
```
<Table
  key={tableKey}
  data={data}
/>
```

Every time when I know this issue will occur, I update this tableKey with an other uuid, which causes the table to rerender with the headers visible. I update this through a useEffect on the change that causes the issue.
